public/manifest.json

```diff
@@ -20,6 +20,6 @@
   ],
   "start_url": ".",
   "display": "standalone",
-  "theme_color": "#000000",
+  "skin_color": "#000000",
```

Comment:
This one isn't related. It should be `theme_color` :)

Reply:
reverted